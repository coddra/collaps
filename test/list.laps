[1, 2, 3] .
["Hello, World!", 1 + 2, [5, 6, 7 * 8]] .
