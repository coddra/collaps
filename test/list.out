[1, 2, 3]
[Hello, World!, 3, [5, 6, 56]]
